### PR TITLE
feat: added explore page andbookmark functionality

### PR DESCRIPTION
**Pull request by anasm11**
Major changes in:
Explore.jsx
Bookmarks.kjsx

### CODE REVIEW

File: src/pages/User.jsx [hunk 1]
```diff
@@ -51,18 +51,39 @@ const User = () => {
...
+                            const base64Url = token.split('.')[1];
+                            const base64 = base64Url.replace('-', '+').replace('_', '/');
+                            const selfname=JSON.parse(window.atob(base64)).username;
+                
+                            const res=await axios({
+                                method:'get',
+                                url:`/api/users`
+                            })
+                
+                            const self=res.data.users.find((user)=>user.username===selfname)
```
Comment (sri-rishi) [inline src/pages/User.jsx:80]: Suggestion: these functions can be extracted in separated function then use that function here

**Review by sri-rishi — APPROVED** [review]
Great work, couldn't find any major issue , feel free to merge

File: src/pages/Profile.jsx [hunk 2]
```diff
@@ -1,57 +1,71 @@
...
         <span className='edit-photo-container bg-gray-300 flex' onClick={(event) => {
-            inputFile.current.click()            
-          }}>
-        
-            <img className='edit-profile-photo' width={'200px'} src={image} />
-            <div className='overlay-text'>Upload new photo</div>
+            inputFile.current.click()
+        }}>
+
+            <img className='edit-profile-photo' width={'200px'} src={image} alt='profile-photo'/>
```
Comment (Naman-Saxena1) [inline src/pages/Profile.jsx]: It would be better/more manageable if you keep all styling in CSS file under classname. Can move width to CSS file

**Review by Naman-Saxena1 — APPROVED** [review]
Overall code looks good, was not able to find any particular issues.
I would suggest you either add guest creds option or give some login creds in PR description, I tried testing functionality on preview and it feels like its not working or something is wrong with it.
So maybe you should cross check that

**Pull request merged**